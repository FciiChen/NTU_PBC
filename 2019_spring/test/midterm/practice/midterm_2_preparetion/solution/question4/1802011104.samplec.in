晚上吃飯的時候，宋盼兒又把今日顧瑾之的表現，活靈活現說給顧延臻聽。

“……出口成章，說的大嫂二嫂和滿屋子的丫鬟婆子都怔愣了！”宋盼兒神采飛揚，“等大嫂的病好了，我要去討份大紅包做診資！”

顧延臻就笑。

他不相信顧瑾之能治好宋大太太。

***

而宋家那邊，正如顧瑾之所料，宋大太太根本沒有讓丫鬟拿著顧瑾之開的方子去抓藥。

宋家專門從錫城請來的那位老大夫叫賴慶坤，在錫城小有名氣。他昨日才到，雖然沒有立刻治好宋大太太，卻也讓她的病症緩解了一刻。如今，賴慶坤叫還在府上，今日又給宋大太太開方子。

宋大太太仍是吃賴慶坤開的藥。

中午的時候，宋大太太肚子不怎麽疼了，可是頭暈得厲害，說話的功夫就眼前冒金花，她躺著歇了一個時辰才好些。

到了傍晚的時候，肚子墜痛，卻無法通便，後綴很重。

宋大太太呻吟著，又叫丫鬟煎了藥來吃。

一劑下去，才微微好了些。

宋大太太身邊的管事孫媽媽就道：“大太太，不如把今日表小姐開的那副方子拿出去，撿了藥來吃？”

宋大太太有氣無力笑了笑：“不中用的！她一個小孩子，懂什麽呢？哄哄她玩兒罷了，你還當真…….”

“可奴婢總聽姑奶奶說，表小姐成日在家裏給人看病，從未失手。”孫媽媽又道。

宋大太太就笑：“這你就不懂了！咱們那位姑奶奶，從小就爭上遊，她女兒有一成好，她都敢誇十成！再說了，就算家裏的丫鬟仆婦被瑾姐兒治壞了，還敢嚷出來不成？定是偷偷另外找大夫吃藥，好了就把功勞算在瑾姐兒身上！”

孫媽媽一想，也對。

顧家是宋盼兒當家，宋盼兒又疼女兒。那些丫鬟仆婦讓顧瑾之治病，無非就是討好夫人和小姐。

治好了，自然使勁誇；治壞了，肯定不會說出來。

所以姑奶奶宋盼兒才說，她家瑾姐兒看病從未失手。

孫媽媽就歎了口氣。

“我瞧著，那位賴大夫也不濟，不如再換一個瞧瞧？”看著宋大太太痛苦的樣子，孫媽媽道。

宋大太太何嚐不知道？

隻是，還上哪裏去請大夫呢？

“算了，治病哪裏是一日就能好的？再吃幾日，等大老爺回來再說吧。”宋大太太輕輕闔了眼簾。

大老爺宋玉在蘇州做鹽法道，已經接到了妻子病重的書信，這幾日在往回趕。

不過片刻，她安睡的模樣一變，眉頭緊緊蹙起來，肚子又開始墜痛了。

宋家三小姐宋言繁來給母親請安，問：“娘，您今日好點了沒有？”

“……好了些。”宋大太太安慰女兒，忍著肚痛笑道。

自從她生病，宋言繁也跟著擔心，人也瘦了一圈，小臉削減了下去。

宋大太太心疼不已。

所以她不準女兒時刻在跟前，隻讓她晨昏定省的時候來坐坐，免得她跟著憂愁。

“是吃了表妹的藥嗎？”宋言繁問母親，“我聽表妹說的頭頭是道，很有學問的樣子！”

她言語裏有點欽佩。

宋言繁是個早產兒，生下來就先天不足。她身體挺好，就是記憶力不行，學什麽都不會。她隻比顧瑾之大兩個月，至今還沒有背會女誡，所以顧瑾之一番說辭讓宋言繁羨慕極了。

“吃了。”宋大太太笑著說。

可是肚子更疼起來。

孫媽媽和丫鬟就服侍她如廁，宋言繁也在一旁幫襯。

半柱香的功夫，宋大太太回來，臉色煞白，又是頭暈目眩，歪倒在**。

宋言繁就嚇哭了，直喊娘。

宋大太太勉強睜開眼，讓人送宋言繁回去。

等宋言繁一走，宋大太太才開始呻|吟。她現在生不如死。

丫鬟們又忙去請了賴慶坤來瞧。

賴慶坤五十來歲，丫鬟們去請的時候，他已經睡下了，迷迷糊糊的跟著來了。他給宋大太太號脈，慢悠悠說著套話。

還是白天一樣的說辭，讓宋大太太再吃幾副藥試試。

宋大太太知道那些藥治標不治本。

人難受的時候，就不那麽理智，她從枕下拿出顧瑾之白天寫的方子給賴慶坤瞧，問他這樣的藥方能不能吃。

賴慶坤借著燭火，眯起眼睛瞧了，然後微怒：“荒唐嘛！看病豈是小兒兒戲？一副藥方才三味藥，他當他是華佗在世？蒼術、升麻、荷葉？這都是什麽亂七八糟的？”

這麽簡單的方子，是哪個孩子寫著玩的嗎？

賴慶坤見宋大太太拿給他瞧，頓起怒火：這是懷疑他的本事，故意試探他？

宋大太太卻眼眸一黯：果然，顧瑾之是學藝不精的。

她的病，是不是沒有指望？

一時間，宋大太太萬念俱灰。她不想死，她的長子還沒有成親，女兒又單純愚笨，沒有她，孩子們怎麽辦？

她疼得厲害，就不顧忌了，當著大夫的麵嗚嗚哭起來。

滿屋子丫鬟婆子都勸她。

賴慶坤也不知所措。

哭了片刻，頭又暈了起來。宋大太太靜養了一會兒，丫鬟就端了藥碗進來。

吃了藥，不知是哭累了還是藥起效果了，宋大太太居然睡了。雖然睡得不沉穩，卻也合眼養神了一會兒。

顧家，宋盼兒問顧瑾之：“你給大舅母的藥，吃幾日能好？”

“一日吃上三劑的話，一日就能好。”顧瑾之道，“隻是不知道大舅母吃了沒有。”

宋盼兒也覺得她的大嫂可能不會吃。

她原本還想去看看病好了沒有，聽顧瑾之這麽一說，就了無興趣了。

隔了兩天，宋盼兒又派了婆子問探病，才知道大嫂的病又重了一層。

她心裏戚戚然，不由想：大嫂這是不中用了嗎？想到這裏，心裏有些發緊，從前對大嫂的各種不滿也消散了，隻擔心她那個人。

也不知道是顧瑾之的藥沒用，還是大嫂根本沒吃？

不管是哪種，都不好告訴顧瑾之。

宋盼兒沒跟顧瑾之提，怕女兒傷心。

到了四月十五，宋家大老爺宋玉終於從蘇州趕了回來，還帶了名蘇州名醫。

宋盼兒聽說宋玉回來了，就又想回娘家去。她從小就跟大哥親。

她還問顧瑾之去不去。

顧瑾之懶懶的，不想去，就說：“近日祖父給了好些功課都沒有做完，我就不去了。”

宋盼兒眉頭蹙了蹙。

她去年給顧瑾之請了位繡娘，教她針黹女紅。可是過了年之後，顧瑾之就推三阻四的，再也沒去學過。

繡娘跟宋盼兒抱怨了數次。

“你不是還要跟程師傅學針黹？”宋盼兒道，“最近怎樣了？”

“過幾日再學吧？”顧瑾之討好著笑，“最近祖父跟我講孔子……”

宋盼兒就板起臉：“你又不下場考狀元，學那些做什麽？當初答應你跟祖父學醫，可是有言在先，你要兼學針黹女紅。不說跟繡娘一樣好，至少也拿得出手。你如今，可有拿得出手的活計？”

顧瑾之就輕撫額頭。

宋盼兒氣不打一處來，還要說她，宋媽媽進來說，馬車已經備好了。

她這才放過了顧瑾之。

顧瑾之回房，原本想溫習《金匱要略》。可想起一方手帕從去年冬月動工，現在才繡了朵寒梅，就讓丫鬟把她的針線簸籮拿出來。

她安靜做著針線，不知不覺到了到了晚飯的時候，母親該從宋家回來了。
BREAK