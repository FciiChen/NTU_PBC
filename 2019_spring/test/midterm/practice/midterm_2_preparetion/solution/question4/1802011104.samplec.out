大_太 28
宋_太 27
顧_之 23
宋_兒 15
宋_繁 9
賴_坤 9
的_候 7
孫_媽 6
的_子 6
了_來 4
